Dockerfile

```
@@ -14,6 +14,8 @@ RUN sed -i.bak "/^#.*deb-src.*universe$/s/^# //g" /etc/apt/sources.list \
     && apt update \
     && apt -y install subversion \
     && apt -y build-dep r-base-dev \
+    && RUN echo "deb https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/" >> /etc/apt/sources.list \
+    && apt update \
     && apt -y install r-base-dev \
     && Rscript -e "install.packages('languageserver', repos='https://cran.rstudio.com')"
```

```suggestion
    && echo "deb https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/" >> /etc/apt/sources.list \
    && apt update \
    && apt -y install subversion \
    && apt -y build-dep r-base-dev \
    && apt -y install r-base-dev \
    && Rscript -e "install.packages('languageserver', repos='https://cran.rstudio.com')"
```

RUN can be removed before echo on line 17. The RUN command starts at line 13 and the \ allows that command to continue on the next line. Moving the echo line above the installation of subversion means we are only running apt update once.